source/FAQ/index.rst

```diff
@@ -86,15 +115,28 @@ bundles whenever possible.
 
 |
 
-Does |CL| use RPMs like other distros?
-======================================
+Does it use RPMs or DEBs packages like other distros?
+=====================================================
+
+No. |CL| provides software to systems in the form of :ref:`bundles-guide`.
+Under the hood, |CL| developers use the RPM format as an intermediary step for
+packaging and determining software dependencies at OS build time.
+
+Individual RPMs and DEBs can sometimes be manually extracted and installed on
+a |CL| system with the right tools, but that is not the intended use case.
+
+|
+
+Why does it have a different approach to software management?
+=============================================================
 
-|CL| provides software in the form of :ref:`bundles-guide`. The RPM
-format is used as an intermediary step for packaging and determining software
-dependencies at OS build time.
+The |CL| team wants software *installation* and *updates* to be as efficient
+and error free as possible. |CL| packages software differently and uses a
+novel updater to solve some of the classic problems with how the software
+packages are on Linux. 
 
-Individual RPMs can sometimes be manually installed on a |CL| system with the
-right tools, but that is not the intended use case.
+For a more detailed explanation, see this discussion on our community forum:
+https://community.clearlinux.org/t/why-does-clearlinux-use-swupd-and-not-apt-deb-rpm/
```

Comment:
Ln 138-139: Embed hyperlink .
For a more detailed explanation, see this `discussion on swupd`_ in our community forum.

Add hyperlink at bottom: 
.. _discussion on swupd: https://community.clearlinux.org/t/why-does-clearlinux-use-swupd-and-not-apt-deb-rpm/3284